example/stretch_grasping/README.md

```diff
@@ -0,0 +1,45 @@
+# Stand-alone grasping setup
+
+## Environment setup on server
+```
+# General installation
+conda create -n home_robot python=3.8 && conda activate home_robot
+conda install -c conda-forge mamba
+pip install -e .
+
+# Specific to stand-alone grasping script
+pip install trimesh pybullet matplotlib open3d opencv-python rospkg numpy==1.21
+git clone https://github.com/mjd3/tracikpy.git && pip install tracikpy
+conda install pytorch==1.11.0 torchvision==0.12.0 torchaudio==0.11.0 cudatoolkit=11.3 -c pytorch
```

Comment:
Might want to add a tip to replace conda with mamba which is optional but strongly recommended (dep resolution goes from ~40 seconds to ~5 seconds).